pages/1.9/release-notes/1.9.7/index.md

```diff
@@ -0,0 +1,215 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.9.7
+title: Release Notes for 1.9.7
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.9.7.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.9.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+# Issues Fixed in DC/OS 1.9.7
+
+- CORE-1375 - Docker executor no longer hangs due to lost messages.
+- DCOS_OSS-1943 - Modified the LIBPROCESS_WORKER_THREADS environment variable to 16 to increase the minimum number of worker threads.
+
+# Notable Changes in DC/OS 1.9.7
+
+- Mesos 1.2.3 is integrated with DC/OS 1.9.7.
```

Comment:
Should this be "has been"? In previous releases (see below, a link to the relevant changelog was provided) and the wording was different.